### PR TITLE
changed css for header to center text

### PR DESCRIPTION
**Pull request by UmbraPlus**
Signed-off-by: Stasioo <59345026+Stasioo@users.noreply.github.com>

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -13,6 +13,7 @@
     margin-bottom: 20px;
 }
 .header {
+    text-align: center;
```
Comment (wcygan) [inline css/styles.css:16]: Please add a picture of what this does 🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠🤠

**Review by wcygan — CHANGES_REQUESTED** [review]
_almost_ LGTM

File: css/styles.css (hunk 1 above)
Comment (Faro0q) [reply css/styles.css:16]: image

**Comment by Faro0q** [comment]
image

**Comment by wcygan** [comment]
great ! ! !

**Review by wcygan — APPROVED** [review]
okie dokie artichokie, looking great!!!

**Pull request merged**